library/std/src/sys/pal/unix/fd.rs

```diff
@@ -120,6 +120,7 @@ impl FileDesc {
         (&mut me).read_to_end(buf)
     }
 
+    #[allow(unused_unsafe)]
```

Comment:
I don't want to make this allowance for OS that don't need it.
```suggestion
    #[cfg_attr(target_os = "vxworks", allow(unused_unsafe)]
```